<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="drawLine">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>20</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="restoreLine">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>70</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Restore Line</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="grid">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>741</width>
      <height>541</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clearAll">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>510</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eraseLine">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>120</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Erase Line</string>
    </property>
   </widget>
   <widget class="QPushButton" name="drawWindow">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>210</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw Clipping Window</string>
    </property>
   </widget>
   <widget class="QPushButton" name="eraseWindow">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>260</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Erase Clipping Window</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clipLineCohenSuther">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>360</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clip Line (Cohen-Sutherland)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clipLineLiangBarsky">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>410</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Clip Line (Liang-Barsky)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1051</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
